src/vos/tests/evt_ctl.c

```diff
@@ -1107,11 +1107,22 @@ test_evt_iter_delete(void **state)
 	int			 sum, expected_sum;
 	struct evt_filter	 filter;
 	uint32_t		 inob;
+	daos_size_t		 scm_used = 0, prev_value = 0;
+	struct umem_instance *um_inst;
+	daos_size_t		initial_value = 0;
+	int		check_um_instance;
 
 	rc = evt_create(EVT_FEAT_DEFAULT, 13, arg->ta_uma, arg->ta_root,
 			DAOS_HDL_INVAL, &toh);
 	assert_int_equal(rc, 0);
-
+	um_inst = utest_utx2umm(arg->ta_utx);
+	check_um_instance = strcmp(um_inst->umm_name, "pmem");
+	if (check_um_instance == 0) {
```

Comment:
Use the umm_id instead of the name.

if (um_inst->umm_id != UMEM_CLASS_VMEM)...

Reply:
Done.